cluster/gce/util.sh

```diff
@@ -912,6 +912,19 @@ function detect-subnetworks() {
   echo "${color_red}Could not find subnetwork with region ${REGION}, network ${NETWORK}, and project ${NETWORK_PROJECT}"
 }
 
+function delete-all-firewall-rules-in-network() {
+  if fws=$(gcloud compute firewall-rules list --project "${NETWORK_PROJECT}" --filter="network=${NETWORK}" --format="value(name)"); then
+    if [[ -z "${fws}" ]]; then
```

Comment:
I think you don't need this if condition. If it is empty string, you'll just pass an empty argument to `delete-firewall-rules()` which will not delete anything inside the for loop.

Reply:
Yep, I was thinking it may be more clear to call it out if there is not firewalls needed to be deleted. But SGTM to remove this if condition.